### PR TITLE
Fix/check sdk version before update#306

### PR DESCRIPTION
**Pull request by alekseytsekov**
verify user's sdk version, if it is higher than predefined one, we will not update it

### CODE REVIEW

File: packages/aeproject-cli/aeproject-init/init.js [hunk 1]
```diff
@@ -59,8 +62,44 @@ const createAEprojectProjectStructure = async (shape) => {
...
+            let updateTo = updateToVersioning[i];
+
+            if (!isNaN(updateTo) && !isNaN(user)) {
+                if (parseInt(user) > parseInt(updateTo)) {
+                    if (await promptUpdate(promptMessage)) {
+                        _sdkVersion = userSdkVersion;
+                        break;
+                    }
+                }
+            } else if (!isNaN(updateTo) && isNaN(user)) {
```
Comment (maptuhec) [inline packages/aeproject-cli/aeproject-init/init.js]: why do you need the **user** to be **nan** here? why in the first place you need the **if** here?

File: packages/aeproject-cli/aeproject-init/init.js [hunk 2]
```diff
@@ -59,8 +62,44 @@ const createAEprojectProjectStructure = async (shape) => {
...
+        let updateToVersioning = _sdkVersion.split('.');
+
+        let promptMessage = `Found newer or different version of sdk ${ userSdkVersion }. Keep it, instead of ${ _sdkVersion }?`;
+
+        for (let i = 0; i < 3; i++) {
+            let user = userVersioning[i];
+            let updateTo = updateToVersioning[i];
+
+            if (!isNaN(updateTo) && !isNaN(user)) {
+                if (parseInt(user) > parseInt(updateTo)) {
```
Comment (maptuhec) [inline packages/aeproject-cli/aeproject-init/init.js]: why comparing only portions of versions but not the whole version instead?

File: packages/aeproject-cli/aeproject-init/init.js (hunk 1 above)
Comment (alekseytsekov) [reply packages/aeproject-cli/aeproject-init/init.js]: if user's version is '5.0.0rc4', '0rc4' can not be parsed and assume that its version is higher

File: packages/aeproject-cli/aeproject-init/init.js (hunk 2 above)
Comment (alekseytsekov) [reply packages/aeproject-cli/aeproject-init/init.js]: how can we compare which of the version is higher 5.0.0 or 6.0.0

File: packages/aeproject-cli/aeproject-init/init.js (hunk 1 above)
Comment (maptuhec) [reply packages/aeproject-cli/aeproject-init/init.js]: Does the npm packages have, **rc** versions?

Comment (alekseytsekov) [reply packages/aeproject-cli/aeproject-init/init.js]: no, but if it is a github link/repo '"package-name": "username/repo#branch-name"'

Comment (maptuhec) [reply packages/aeproject-cli/aeproject-init/init.js]: valid point. In my opinion you can remove the **!isNaN(updateTo)** from both **if** statements and then change the **else if** to only else

**Pull request merged**